attributes/default.rb

```diff
@@ -30,8 +30,8 @@
 #   default port: 5601, 80
 
 # User and group on the server the application is being deployed
-default['mconf-stats']['user']      = node['mconf']['user'] || 'mconf'
-default['mconf-stats']['app_group'] = node['mconf']['app_group'] || 'www-data'
+default['mconf-stats']['user']      = node['mconf-stats']['user'] || 'mconf'
```

Comment:
This was actually right as it was before. The idea is that it will try to use the attributes from `node["mconf"]` if they are set. Otherwise it will set a default value.
But above these defaults, if the user sets the attributes directly on `node['mconf-stats']` than these are the ones that will be used.